packages/react/src/components/table/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Table';
+export * from './DataTable';
```

Comment:
Not really sure how the exporting of multiple components work here. The idea was that people would be able to compose their own table from the components that the `Table` component provides (look at the end of Table component file for more info). But also we would expose the actual complete Table, named DataTable here. Im not sure if we need/want people to be able to compose their own table from sub-components of Table? Would DataTable be enough already?

Reply:
Can Table be used ad standalone, or does it always require having DataTable around it? If it always requires DataTable anyway, it might be enough. If Table works alone, it should be available separately.

Reply:
Table works without DataTable, but then you need to add other table contents to it. Table basically just wraps native html table with hds styles to it. We discussed renaming it to TableContainer, so that we could then rename DataTable to just Table.

Reply:
We decided to publish only Table now (former DataTable), and then publish sub-components later on if they are needed / wanted.